### ISSUE TITLE
Adding a file resource to a resx doesn't work as expected

### ISSUE THREAD

**Issue by piotrpMSFT** [aspnet/Tooling#559]
## Steps to reproduce

Create a new project, class library (.netcore).
Add a Resources.resx to the project.
Add a file TestResource.txt to the file, containing "test text".
Open the Resources.resx file.
Select Files from the resource type dropdown.
Drag the TestResource.txt file into the resx designer.
Compile.
Inspect the compiled assembly (ilspy).
## Expected  behavior

The compiled resource contains a name = "TestResource" with value = "test text".
## Actual behavior

The compiled resource contains a name = "TestResource" with value = "TestResource.txt;System.String, mscorlib, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089;utf-8"
## Environment data

`dotnet --info` output:

.NET Command Line Tools (1.0.0-preview1-002702)

Product Information:
 Version:     1.0.0-preview1-002702
 Commit Sha:  6cde21225e

Runtime Environment:
 OS Name:     Windows
 OS Version:  10.0.10586
 OS Platform: Windows
 RID:         win10-x64

/ported from https://github.com/dotnet/cli/issues/3383 on behalf of @Markpflug

**Comment by sayedihashimi** [comment]
TFS Bug#<a href="https://devdiv.visualstudio.com/DefaultCollection/DevDiv/_workItems#id=231148&fullScreen=false&_a=edit">231148</a>

**Comment by sayedihashimi** [comment]
Sorry for the delay here. We've had a lot of updates over the past few months and I believe that this is working now in VS2017 RC. Can you try with the latest VS2017 and try again? I'll close this now, but please reopen if this issue still persists.

**Comment by MarkPflug** [comment]
This issue appears to be fixed. Having said that...

The generated code for Resource.resx contains: 
global::System.Resources.ResourceManager("ClassLibrary1.Resource1", typeof(Resource1).Assembly);

Which doesn't compile, because Type.Assembly doesn't exist (netstandard 1.4). Is this expected? I commented out the offending code to make things compile.

It appears that ILSpy can't open this assembly any more. Sad day, but not your problem. I opened the dll in notepad and saw the resource string instead of the filename, so I assume *that* part of it is now working correctly.

I'll open a new issue for the code-generation issue.
